src/CarbonAware.DataSources/CarbonAware.DataSources.ElectricityMaps/src/Client/IElectricityMapsClient.cs

```diff
@@ -0,0 +1,45 @@
+using CarbonAware.DataSources.ElectricityMaps.Model;
+
+namespace CarbonAware.DataSources.ElectricityMaps.Client;
+
+/// <summary>
+/// An interface for interacting with the Electricity Maps API.
+/// </summary>
+public interface IElectricityMapsClient
+{
+    public const string NamedClient = "ElectricityMapsClient";
+
+    /// <summary>
+    /// Async method to get the most recent 24 hour forecasted emission data for a given latitude and longitude.
+    /// </summary>
+    /// <param name="latitude">Latitude for query</param>
+    /// <param name="longitude">Longitude for query</param>
+    /// <returns>A <see cref="Task{ForecastedCarbonIntensityData}"/> which contains forecasted emissions data points.</returns>
+    /// <exception cref="ElectricityMapsClientException">Can be thrown when errors occur connecting to ElectricityMaps client.  See the ElectricityMapsClientException class for documentation of expected status codes.</exception>
+    public Task<ForecastedCarbonIntensityData> GetCurrentForecastAsync(string latitude, string longitude);
```

Comment:
We could do that, it was more for this to match to similar functions in WattTime Client so that the mental model of what the function is doing can map between clients. Don't feel super strongly, more just giving context of what the thinking was behind that naming scheme.

Reply:
Post convo with Bill @JenMadiedo , will update to matchc better to EMaps